dags/openshift_nightlies/scripts/install/rosa.sh

```diff
@@ -24,6 +24,26 @@ _get_cluster_status(){
     echo $(rosa list clusters -o json | jq -r '.[] | select(.name == '\"$1\"') | .status.state')
 }
 
+_wait_for_cluster_ready(){
+    echo "INFO: Waiting about 1 hour for cluster on ready status"
+    # 180 iterations, sleeping 60 seconds, 3 hours of wait
+    for i in `seq 1 180` ; do
+        CLUSTER_STATUS=$(_get_cluster_status $1)
+        if [ ${CLUSTER_STATUS} == "ready" ] ; then
+            sleep 120 # Wait 60 seconds to allow OCM to really finish the cluster installation
```

Comment:
Comment is different than the sleep command

Reply:
Updated all comments to new sleep values and counters